packages/driver/cypress/integration/e2e/multi-domain/navigation_spec.ts

```diff
@@ -78,16 +95,26 @@ describe('navigation events', { experimentalSessionSupport: true, experimentalMu
         })
 
         cy.get('button[data-cy="reload"]').click()
-        cy.wrap(p)
-      }).then(() => {
-        assertLogLength(logs, 14)
-        expect(logs[10].get('name')).to.eq('page load')
+        cy.wrap(afterWindowLoad).then(() => {
+          return logs.map((log) => ({ name: log.get('name') }))
+        })
+      })
+      .then(reifyLogs)
+      .then((secondaryLogs) => {
```

Comment:
Logs in the secondary no longer have a counterpart in the primary, so these tests needed to change accordingly.